src/Kros.AspNetCore/Authorization/JwtAuthorizationHelper.cs

```diff
@@ -27,6 +31,11 @@ public static class JwtAuthorizationHelper
         /// </summary>
         public const string OAuthSchemeName = "OAuthAuthorization";
 
+        /// <summary>
+        /// Authorization token prefix.
+        /// </summary>
+        public static string AuthTokenPrefix = "Bearer";
```

Comment:
Má nejaký význam prečo je to `static` a nie `const`?

Reply:
opravene